viewer/packages/360-images/src/icons/Image360PointsObject.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { OverlayPointsObject } from '@reveal/3d-overlays';
+import { PointsMaterialParameters } from '@reveal/3d-overlays/src/OverlayPointsObject';
```

Comment:
Need to export `PointsMaterialParameters`?

Reply:
Yeah, since it is used in `OverlayPointsObjects` parameters.

Reply:
Right, I just meant - should we export it from `3d-overlays`, so that we don't need to import it directly through the file path?